apps/server/src/v1/monitors/post.ts

```diff
@@ -67,18 +74,33 @@ export function registerPostMonitor(api: typeof monitorsApi) {
       throw new HTTPException(403, { message: "Forbidden" });
     }
 
-    const { headers, regions, ...rest } = input;
+    const { headers, regions, assertions, ...rest } = input;
+
+    const assert: Assertion[] = [];
+    if (assertions) {
+      for (const a of assertions) {
+        if (a.type === "header") {
+          assert.push(new HeaderAssertion({ ...a, version: "v1" }));
+        }
+        if (a.type === "textBody") {
+          assert.push(new TextBodyAssertion({ ...a, version: "v1" }));
+        }
+        if (a.type === "status") {
+          assert.push(new StatusAssertion({ ...a, version: "v1" }));
+        }
+      }
+    }
```

Comment:
Lets just use the `import {deserialize} from '@openstatus/assertions'` function on that. It already does what we need.

Reply:
no because we need to add the version :(

Reply:
and assertion from the payload is already an assertion but we don't push the version